packages/features/sales-escrows/pages/NFTPage/index.tsx

```diff
@@ -27,13 +27,74 @@ import {
   Banner,
   TabContent,
   ShowMoreBlock,
+  theme,
 } from '@origyn-sa/origyn-art-ui';
 import { useDialog } from '@connect2ic/react';
 import { getNftCollectionMeta, OrigynClient } from '@origyn-sa/mintjs';
 import { EscrowType } from '../../modals/StartEscrowModal';
 import { Principal } from '@dfinity/principal';
 import { PlaceholderImage } from '@dapp/common-assets';
 
+export interface CheckOfferProps {
+  odc: OdcDataWithSale;
+  onOpenEscrowModal: (escrowType: EscrowType) => void;
+}
+
+export const CheckOffer = ({ odc, onOpenEscrowModal }: CheckOfferProps) => {
```

Comment:
Please put this in it's own file.

Reply:
CheckOffer is an action, not the best name for a component. Components should be named as nouns. "OffersPanel" would work, or something similar.

Reply:
Done